misc/config/config.yaml.tmpl

```diff
@@ -0,0 +1,46 @@
+# Configuration file of Harbor Acceleration Service
+
+# http related config
+server:
+  name: API
+  # listened host for http
+  host: 0.0.0.0
+  # port for http
+  port: 2077
+
+metric:
+  # export metrics on `/metrics` endpoint
+  enabled: true
+
+provider:
+  source:
+    # hostname of harbor service
+    hub.harbor.com:
+      # base64 encoded `<robot-name>:<robot-secret>` for robot
+      # account created in harbor
+      auth: YTpiCg==
+      # use http registry communication
+      insecure: true
+      webhook:
+        # webhook request auth header configured in harbor
+        auth_header: header
+  containerd:
+    # ensure containerd service listening on this address
+    address: /run/containerd/containerd.sock
+    snapshotter: overlayfs
+
+converter:
+  # number of worker for executing conversion task
+  worker: 5
+  driver:
+    # accelerator driver type: `nydus`
+    # estargz driver implementation is in progress
+    type: nydus
+    config:
+      work_dir: /tmp
+      # `nydus-image` binary path, download it from:
+      # https://github.com/imeoer/image-service/releases/download/v1.0.0/nydus-image
+      builder: nydus-image
```

Comment:
Is it possible to enable multiple drivers? Do we need to run multiple acceleration-service processes with different configurations to enable multiple converters?

Reply:
``` yaml
  drivers:
    - type: nydus
      config:
        # ...
      rules:
          # source image reference that matches the regex will be converted
        - match: ns1/*
          tag_suffix: -converted
    - type: estargz
      config:
        # ...
      rules:
        - match: ns2/*
          tag_suffix: -converted
```

Should be a possible scenario (it's not difficult to support), how about this format? The source image will be converted by the first matched rule.